### PR TITLE
- adds auto-increment of the PHP version

### PR DESCRIPTION
**Pull request by baywet**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoftgraph/MSGraph-SDK-Code-Generator/pull/1211)

### CODE REVIEW

File: .azure-pipelines/generation-templates/php-kiota.yml [hunk 1]
```diff
@@ -9,3 +9,12 @@ steps:
     BuildConfiguration: $(buildConfiguration)
     OutputFullPath: $(kiotaDirectory)/output/*
     RepoModelsDir: $(Build.SourcesDirectory)/${{ parameters.repoName }}/src/Generated/
+
+# right now the script runs fine with PHP 8.1 which is what is installed by default on the agent
+# https://github.com/actions/runner-images/blob/main/images/ubuntu/Ubuntu2204-Readme.md
+# but we should use a setup task to install the right version of PHP
+# however, there does not seem to exist a setup-php task (equivalent to UseDotnet)
+
+- pwsh: 'php $(Build.SourcesDirectory)/${{ parameters.repoName }}/scripts/BumpStableSdkVersion.ps1'
```
Comment (Ndiritu) [inline .azure-pipelines/generation-templates/php-kiota.yml:18]:
```suggestion
- pwsh: 'php $(Build.SourcesDirectory)/${{ parameters.repoName }}/scripts/BumpStableSdkVersion.php'
```

**Review by Ndiritu — CHANGES_REQUESTED** [review]

File: .azure-pipelines/generation-templates/php-kiota.yml (hunk 1 above)
Comment (Ndiritu) [reply .azure-pipelines/generation-templates/php-kiota.yml:18]: Would it be worthwhile to log an issue to streamline all scripts to PowerShell?

Comment (andrueastman) [reply .azure-pipelines/generation-templates/php-kiota.yml:18]: I think the structure of the different project files are different across languages ranging from xml/json formats, if a given language is having scenario that is easiest solved in a certain way that isn't powershell, I think it makes sense to go that way.

**Review by andrueastman — COMMENTED** [review]
Looks good to me. 
(Not approving for now so that @Ndiritu can review before merge.)

**Review by Ndiritu — APPROVED** [review]

**Pull request merged**